code/modules/paperwork/pen.dm

```diff
@@ -147,8 +147,20 @@
 	var/on = FALSE
 	var/brightness_on = 2
 	light_color = LIGHT_COLOR_RED
+	stealthy_audio = TRUE
 	armour_penetration_flat = 20
 
+/obj/item/pen/edagger/attack(mob/living/M, mob/living/user, def_zone)
+	if(on && user.dir == M.dir && !HAS_TRAIT(src, TRAIT_FLOORED) && user != M)
```

Comment:
Think you meant to check that the user or M is floored? Right now you're checking if the edagger is floored.